### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jonathanmunamire**
### Project requirements

- You should follow the layout of the wireframes provided. You should personalize the rest of the design including colors, typographies, spacings, etc.
- Login page and registration page:
    - Should be built with Devise.
- Food list:
    - Should display a list of food added by the logged-in user as in the wireframe (for teams with only 2 people - display also quantity of a given food).
    - Should lead to a form that allows users to add new food.
- Recipes list:
    - Should display a list of recipes created by the logged-in user as in the wireframe.
    - Should lead to recipe details.
    - If the user is the owner of the recipe, should allow the user to delete it.
- Public recipe list:
    - Should display a list of all public recipes ordered by newest as in the wireframe.
    - Should lead to recipe details.
- Recipe details:
    - Should display a toggle button that allows for a recipe to be made public or private.
    - If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
    - If the user is the owner of the recipe, should lead to the form that allows the user to add new food.
- General shopping list view (**only for a group that has less than 3 members**):
    - Should show the list of food that is missing for all recipes of the logged-in user (compare the list of food for all recipes with the general food list of that user).
    - Should count the total food items and total price of the missing food.
- Make sure there are no N+1 queries happening.
- Create a navigation menu that allows users to open all of the pages you created.
- Write unit and integration tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,154 @@
...
+If you like this project...give it a ⭐️!
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [@microverse](https://www.microverse.org/)
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/jonathanmunamire/Recipe_App/blob/e8aa46d19e1f0e413df26a25a6eaa1b0f1e28068/LICENSE) licensed.
+
```
Comment (peterrobert) [inline README.md:151]: - [ ] The link to the MIT license is broken. It is better to have a link that goes to the MIT license. Kindly fix this issue

File: README.md [hunk 2]
```diff
@@ -1,24 +1,154 @@
...
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add a meal planner to help users plan their weekly meals.
+- Allow users to add pictures to their recipes.
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Ajonathanmunamire/Recipe_App.git/issues).
```
Comment (peterrobert) [inline README.md:138]: - [ ] The link to the issue is broken. It is better to have a link that goes to the issue. Kindly fix this issue

**Review by peterrobert — CHANGES_REQUESTED** [review]
Hi @jonathanmunamire  and @Ibnballo1 , 

Very nice project. I love the way you have styled your application it looks very nice 👏🏿 . However, there are a few changes to be made kindly check below ⬇️

### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️

 ## Required Changes ♻️
_Check the comments under the review._

### N+1 PROBLEM

- [ ] It will be great to eliminate all N+1 problems in your application. The best way to detect this issue is to use a gem called bullet gem you can find it [here](https://github.com/flyerhzm/bullet) just install it in your application and use it in the development environment to detect and eliminate all the N+1 problems. Some of the areas I have detected this problem are: 

1. When navigating to the general shopping list. 
2. When navigating to the public recipe list.
3. When generating a shopping list.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi team 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Good user interface
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**